spec/index.html

```diff
@@ -736,8 +730,31 @@ <h4>sd:SPARQL11Query</h4>
         </section>
         <section id="lang-sparql11update">
           <h4>sd:SPARQL11Update</h4>
-          <p><code>sd:SPARQLUpdate</code> is an <a href="#sd-Language"><code>sd:Language</code></a> representing the <a href="http://www.w3.org/TR/sparql11-update/">SPARQL 1.1 Update language</a>
-          [<a href="#UPDATE11">UPDATE11</a>].</p>
+          <p><code>sd:SPARQL11Update</code> is an <a href="#sd-Language"><code>sd:Language</code></a> representing the [[[SPARQL11-UPDATE]]] language.</p>
+          <table>
+            <tbody>
+              <tr>
+                <th>type:</th>
+                <td><a href="#sd-Language"><code>sd:Language</code></a></td>
+              </tr>
+            </tbody>
+          </table>
+        </section>
+        <section id="lang-sparql11query">
+          <h4>sd:SPARQL12Query</h4>
+          <p><code>sd:SPARQL12Query</code> is an <a href="#sd-Language"><code>sd:Language</code></a> representing the [[[SPARQL12-QUERY]]].</p>
+          <table>
+            <tbody>
+              <tr>
+                <th>type:</th>
+                <td><a href="#sd-Language"><code>sd:Language</code></a></td>
+              </tr>
+            </tbody>
+          </table>
+        </section>
+        <section id="lang-sparql11update">
```

Comment:
```suggestion
        <section id="lang-sparql12update">
```
(These two id fixes shoudl make Echnida happy)